src/common/Error/styled.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+import { ReactComponent as Error } from "./error.svg";
+
+export const Wrapper = styled.div`
+  margin: 195px auto 0 auto;
+  text-align: center;
+  height: 100vh;
```

Comment:
this `height: 100vh` is unnecessary, better use margin-bottom like I suggested to separate from the bottom of the page.

Reply:
ok i corrected it